samples/CommunityToolkit.Maui.Sample/Pages/Converters/ImageResourceConverterPage.xaml

```diff
@@ -2,17 +2,30 @@
 <pages:BasePage 
     xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
     xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    xmlns:mct="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
     xmlns:pages="clr-namespace:CommunityToolkit.Maui.Sample.Pages"
     x:Class="CommunityToolkit.Maui.Sample.Pages.Converters.ImageResourceConverterPage"
     xmlns:vm="clr-namespace:CommunityToolkit.Maui.Sample.ViewModels.Converters"
     x:TypeArguments="vm:ImageResourceConverterViewModel"
     x:DataType="vm:ImageResourceConverterViewModel">
 
+    <ContentPage.Resources>
+        <ResourceDictionary>
+            <mct:ImageResourceConverter x:Key="ImageResourceConverter" />
+        </ResourceDictionary>
+    </ContentPage.Resources>
+    
     <pages:BasePage.Content>
-        <VerticalStackLayout>
-            <Label Text="ImageResourceConverter"
-                VerticalOptions="CenterAndExpand" 
-                HorizontalOptions="CenterAndExpand" />
-        </VerticalStackLayout>
+        <Grid RowDefinitions="100, *" ColumnDefinitions="*,*,*">
```

Comment:
We don't need to define the `ColumDefinitions` if all are `*`, because it's the default value
```suggestion
        <Grid RowDefinitions="100, *">
```